### DISCUSSION TITLE
Problema Workflow Autograding

### DISCUSSION THREAD

**Discussion by MiltonMartorello** [dds-jv/foro#318] [Consultas de herramientas] [answered]
Buenas noches. tuve algunos problemas para verificar el TPI4 de persistencia

El mvn verify corre bien
![image](https://user-images.githubusercontent.com/49041770/135736001-4146b7f3-6152-446c-81af-1f21be5f5605.png)

me llega este mail cuando pusheo con el tag entrega-final
![image](https://user-images.githubusercontent.com/49041770/135736079-f3eafbf4-b9fe-4efd-9fb6-2064005121eb.png)

Borre el tag y lo volvi a crear pusheando. Se corrió bien el workflow de autograding pero no me aparece el check del TP. Es un bug ?
![image](https://user-images.githubusercontent.com/49041770/135736182-e15e2558-5bf3-464b-a910-00bec4117005.png)
![image](https://user-images.githubusercontent.com/49041770/135736188-0d035567-90f8-4626-a464-8f4b5dca244e.png)

Gracias, saludos

**Comment by gastonprieto** [comment]
ni idea, tal vez se queda con el primero que ejecutaste, pero tal vez @flbulgarelli sabe.

**Comment by flbulgarelli** [comment, marked as answer]
Hola, @MiltonMartorello. 

Github no te está poniendo la tilde verde porque el tag que hiciste no se para sobre tu rama principal (`main`). Para que te lo tome, tenés que crear el tag desde `main`, y asegurarte también de _pushear_ todos los cambios en esa rama a github también. 

En otras palabras, si entrás acá https://github.com/dds-utn/tpi4-juego-de-tronos-MiltonMartorello/, deberías ver exactamente lo mismo que acá https://github.com/dds-utn/tpi4-juego-de-tronos-MiltonMartorello/tree/entrega-final. Notá que los hashes de los commits en este momento no coinciden, pero deberían.
